<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="editDataButton">
        <property name="text">
         <string>EDIT DATASET</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dataButton">
        <property name="text">
         <string>DATA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="infoButton">
        <property name="text">
         <string>INFORMATION</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDownload_CSV"/>
    <addaction name="actionUpload_CSV"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionView_logo"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionDownload_CSV">
   <property name="text">
    <string>Download CSV</string>
   </property>
  </action>
  <action name="actionView_logo">
   <property name="text">
    <string>View logo</string>
   </property>
  </action>
  <action name="actionUpload_CSV">
   <property name="text">
    <string>Upload CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
